src/ResourceManager/Profile/Commands.Profile/Environment/AddAzureRMEnvironment.cs

```diff
@@ -285,6 +290,8 @@ public override void ExecuteCmdlet()
                                    nameof(GraphAudience));
                                 SetEndpointIfBound(newEnvironment, AzureEnvironment.Endpoint.DataLakeEndpointResourceId,
                                     nameof(DataLakeAudience));
+                                SetEndpointIfBound(newEnvironment, AzureEnvironment.Endpoint.BatchEndpointResourceId,
```

Comment:
Note, I think you will also need to change the constructors and type conversion operators for PSAzureEnvinronment to include this value.  If you add this value into existing test validation overt these types, you'll see the code you need to change.

Reply:
This was already done I think, right? I made the change to `PSAzureEnvironment(PSObject other)` already, and to `Equals` in `PSAzureEnvironment` - Did I need to do something else too?